pkgs/tools/networking/tnat64/default.nix

```
@@ -0,0 +1,36 @@
+{ lib, stdenv, fetchFromGitHub, autoreconfHook }:
```

```suggestion
{ lib
, stdenv
, fetchFromGitHub
, autoreconfHook
}:
```
Standard is one line per

---

It's fine if there are only so few